src/optimizer/util.cpp

```diff
@@ -23,6 +26,145 @@ namespace peloton {
 namespace optimizer {
 namespace util {
 
+void FillTransitiveTable(
```

Comment:
i think this method should have some inline documentation about what is happening here